starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -7,3 +7,30 @@ html {
   font-family: 'Open Sans', sans-serif;
   font-size: 18px
 }
+body {
+  color: rgb(84, 86, 89);
+  
+}
+.div1 {
+  background-image: url("/Users/levijskal/Desktop/Ironhack/Class-1/assigment2/lab-css-recipes-clone/starter-code/french-onion-soup-starter-code/images/soup.jpg");
```

Comment:
just like the apple pie image for the path, you would path like this `url("../images/soup.jpg")`